ophyd/pv_positioner.py

```diff
@@ -105,9 +102,9 @@ def __init__(self, prefix='', *, settle_time=0.05, limits=None, name=None,
             raise ValueError('A setpoint or a readback must be specified')
 
         if self.done is None and not self.put_complete:
-            msg = ('PVPositioner %s is mis-configured. A "done" Signal must be '
-                   'provided or use PVPositionerPC (which uses put completion '
-                   'to determine when motion has completed).'
+            msg = ('PVPositioner %s is mis-configured. A "done" Signal must be'
```

Comment:
Ugh, this is why I'm worried about the `f''` strings in 3.6... I already can't keep them straight!